client/lib/plans/features-list.js

```diff
@@ -1073,7 +1087,7 @@ export const FEATURES_LIST = {
 
 	[ FEATURE_PRODUCT_SEARCH_V2 ]: {
 		getSlug: () => FEATURE_PRODUCT_SEARCH_V2,
-		getTitle: () => i18n.translate( 'Site Search: up to 100k records' ),
+		getTitle: () => i18n.translate( 'Search up to 100k records' ),
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.